.github/dependabot.yml

```diff
@@ -3,38 +3,16 @@
 
 version: 2
 updates:
-  - package-ecosystem: "bundler"
-    directory: "/dev/ci/mac"
-    schedule:
-      interval: "weekly"
-    reviewers:
-      - "christopherfujino"
-      - "jmagman"
-      - "keyonghan"
-    labels:
-      - "team"
-      - "team-infra"
-      - "autosubmit"
-  - package-ecosystem: "docker"
-    directory: "/dev/ci/docker_linux"
-    schedule:
-      interval: "weekly"
-    reviewers:
-      - "christopherfujino"
-      - "jmagman"
-      - "keyonghan"
-    labels:
-      - "team"
-      - "team-infra"
-      - "autosubmit"
   - package-ecosystem: "github-actions"
     directory: "/"
```

Comment:
What does this entry do? Run all github actions? Will this mean the labeler and release workflows will run only weekly?

Reply:
No, this `.github/dependabot.yml` config is telling dependabot to update the `uses` dependencies needed to run github actions.  

For example, dependabot opened https://github.com/flutter/flutter/pull/156366 and updated all the `.github/workflows/` yaml files to point to the latest version of `actions/checkout`
https://github.com/flutter/flutter/blob/c36ad608da8ccc0be9c4ee95df4eb594c8aa46b2/.github/workflows/coverage.yml#L22

This PR:
1. Groups all the dependabot updates for github-actions into one PR.  For example, if there were an update for `actions/checkout` and `codecov/codecov-action`, dependabot will send out one PR updated both instead of one PR per dependency.
https://github.com/flutter/flutter/blob/c36ad608da8ccc0be9c4ee95df4eb594c8aa46b2/.github/workflows/coverage.yml#L22-L26
2. Changes this dependabot version check from happening daily to happening weekly.

Reply:
Ahh ok, thanks for the clarification.